src/components/__tests__/AutocompleteInput.spec.ts

```diff
@@ -1,15 +1,15 @@
 import AutocompleteInput from '@/components/AutocompleteInput.vue'
 import {beforeEach, describe, expect, jest, test} from '@jest/globals'
-import {mount, Wrapper, WrapperArray} from '@vue/test-utils'
+import {mount, VueWrapper} from '@vue/test-utils'
 import Vue from 'vue'
 import axios from 'axios'
 import * as bulmaToast from 'bulma-toast'
 import {Options} from 'bulma-toast'
 
 jest.mock('axios')
 
-describe('AutocompleteInput.vue', () => {
-  let wrapper: Wrapper<Vue>
+describe('AutocompleteInput', () => {
+  let wrapper = mount(AutocompleteInput)
```

Comment:
We do the actual mount on line 37 during `beforeEach`

maybe this? 

```suggestion
  let wrapper: VueWrapper<AutocompleteInput>
```

or this

```suggestion
  let wrapper: VueWrapper
```


Repeat for all other spec files as well